### ISSUE TITLE
Make a comment here if you're at HACK CFIN 1 (Wed 8)

### ISSUE THREAD

**Issue by cjayb** [meeg-cfin/sandbox#20]
Hi!

**Comment by MadsJensen** [comment]
moijn

**Comment by Chrichristelle** [comment]
Hello

**Comment by sarathykousik** [comment]
heyloo

**Comment by ceciliemoeller** [comment]
davs

**Comment by patriciamota** [comment]
Hi!!

**Comment by yurysh** [comment]
belated comment from the corner....

**Comment by vukovicnikola** [comment]
Hello!

**Comment by martinjdietz** [comment]
Hola

**Comment by sarangnemo** [comment]
yo

**Comment by RashaHyder** [comment]
Finally hi

**Comment by ahoejlund** [comment]
Very belated hi

**Comment by vukovicnikola** [comment]
This wiki page may be useful as a template to those developing the analyses and code pages: https://wiki.cimec.unitn.it/tiki-index.php?page=DataAnalysisMain

**Comment by MerellaTheMushroomQueen** [comment]
Hejsa!

**Comment by vukovicnikola** [comment]
This is for new people coming to CFIN - would it be useful to have a page on the wiki which is illustrated with pictures of the equipment and lab setup (e.g. annotations of what all those buttons in the MEG computing rack do, TMS/EEG connectors, consumables, etc.)?

**Comment by sarathykousik** [comment]
Yeah. Like a pic-manual. I'm already jittery after the IHR system installation. :P

**Comment by cjayb** [comment]
@vukovicnikola @sarathykousik This is on me: I'll be working on MEG and EEG lab operating procedures over the summer (as I make major changes to the setup there now...)
